code/game/objects/items/circuitboards/machines/machine_circuitboards.dm

```diff
@@ -149,7 +149,8 @@
 	req_components = list(
 		/datum/stock_part/servo = 2,
 		/obj/item/stack/cable_coil = 1,
-		/datum/stock_part/filter = 1)
+		/datum/stock_part/filter = 1
+		)
```

Comment:
```suggestion
		/datum/stock_part/filter = 1,
	)
```

there's a very specific way to multiline lists, and that's that you need both a trailing comma and the indentation level of the closing parenthesis must match the starting indentation of the list variable

Reply:
what the, is a filter none of my machines use that

Reply:
oh lol tcomms parts okay